packages/storefront-lwc/src/modules/commerce/productDetail/productDetail.html

```diff
@@ -93,6 +93,19 @@ <h3 class="float-left hidden-sm-down">{product.name}</h3>
                     </div>
                 </div>
 
+                <!-- Product Recommendations -->
+                <template if:true={product.recommendations}>
```

Comment:
does this return false for an empty array?

Reply:
false